src/liveBlock.ts

```diff
@@ -21,6 +30,26 @@ type LiveBlock = {
 
 // ----- Functions ----- //
 
+const serialiseLiveBlock = ({
+    id,
+    isKeyEvent,
+    title,
+    firstPublished,
+    lastModified,
+    body,
+}: LiveBlock): JsonSerialisable =>
+    ({
+        id,
+        isKeyEvent,
+        title,
+        firstPublished: optionToSerialisable(firstPublished),
```

Comment:
Isn't the fact we have to pass options to a special function a smell that we'd be better off without the `Option` type?
Could `MyType | undefined | null` suffice?

Reply:
I feel the `Option` type offers us a lot of benefits that I wouldn't want to throw away for the sake of avoiding a bit of work to serialise them.

What this says to me is that some of structures we use to represent our data don't fit into the subset of types that count as valid JSON (`Option` isn't the only example of this). I don't think that should force us to lose the richness of our data modelling by mandating 1:1 JSON compatibility.

Reply:
I wasn't aware of the `replacer` feature, that might be a good candidate?

>I feel the Option type offers us a lot of benefits that I wouldn't want to throw away for the sake of avoiding a bit of work to serialise them.

That's fair!

Reply:
As an aside, this problem might go away if we were to move away from classes and methods to data types and pure functions. See [this related discussion](https://github.com/guardian/types/issues/5#issuecomment-632045893) that's also going on today.